src/sentry_plugins/heroku/__init__.py

```diff
@@ -1,5 +1 @@
 from __future__ import absolute_import
-
-from sentry_plugins.base import assert_package_not_installed
-
-assert_package_not_installed("sentry-heroku")
```

Comment:
@joshuarli Removing these is totally fine

Reply:
Perfect, thanks.

Reply:
Decoupled into https://github.com/getsentry/sentry/pull/18628.